actors/builtin/power/power_actor.go

```diff
@@ -321,6 +322,20 @@ func (a Actor) CurrentTotalPower(rt Runtime, _ *abi.EmptyValue) *CurrentTotalPow
 	}
 }
 
+// Fails if the caller does not have a claim.
+// ErrIllegalState signals an internal error in the power actor claim table
+// ErrForbidden indicates the claim is not found in claims table
+func (a Actor) CallerHasClaim(rt Runtime, _ *abi.EmptyValue) *abi.EmptyValue {
+	rt.ValidateImmediateCallerAcceptAny()
+	minerAddr := rt.Caller()
+
+	var st State
+	rt.StateReadonly(&st)
+
+	validateMinerHasClaim(rt, st, minerAddr)
+	return nil
```

Comment:
I know you only need the boolean here, but it seems shortsighted not to return a useful value here, like some claim information. Otherwise we're as likely as not to need to add another method for that in the future.